src/folketrygdloven_kap2_bestemmelser.js

```diff
@@ -4,8 +4,24 @@
  */
 const folketrygdloven_kap2_bestemmelser = [
   {
-    kode: 'FTRL_KAP2_2_1',
-    term: null
+    kode: 'FTRL_KAP2_2_1_FØRSTE_LEDD',
```

Comment:
Catchet ikke helt at det var endringer i eksisterende koder her, men da må du nok inn å fikse (iallefall) api med nytt kdoeverk. Og vurdere om du endrer databasen. Vet ikke om det trengs siden det ikke er prodsatt enda, blir isåfall bare i dev 🤷‍♀️

Reply:
@thanglyphan ☝️

Reply:
Yep, jeg skal se på det